corehq/tests/nose.py

```diff
@@ -177,12 +178,15 @@ class HqdbContext(DatabaseContext):
       rebuilt.
     - `REUSE_DB=teardown` : skip database setup; do normal teardown after
       running tests.
+    - `REUSE_DB=migrate` : skip normal database setup except for running
+      migrations.
```

Comment:
This is ambiguous about teardown. What do you think of "same as \`REUSE_DB=1\` except migrate databases before running tests."?